internal/sessioncmd/terminal.go

```
@@ -179,12 +207,58 @@ func (t *Terminals) Create(sessionID string, opts CreateTerminalOptions) (Termin
+	}
 	_ = runtime.driver.SetLabel(sess.ID, sessionLabel(sess.Group, sess.Name))
-	return runtime.info(sess, true), nil
+	return runtime.info(sess, true)
+}
+
+func (t *Terminals) Close(sessionID, terminalID string) error {
+	runtime, err := t.open()
+	if err != nil {
+		return err
+	}
+	defer runtime.store.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `runtime.store.Close` is not checked

(errcheck)